packages/datadog-plugin-tedious/test/index.spec.js

```diff
@@ -0,0 +1,143 @@
+'use strict'
+
+const agent = require('../../dd-trace/test/plugins/agent')
+const plugin = require('../src')
+
+const MSSQL_USERNAME = 'sa'
+const MSSQL_PASSWORD = 'DD_HUNTER2'
+
+wrapIt()
+
+describe('Plugin', () => {
+  let tds
+  let tracer
+  let connection
+
+  withVersions(plugin, 'tedious', version => {
+    beforeEach(() => {
+      tracer = require('../../dd-trace')
+    })
+
+    describe('without configuration', () => {
+      let config
+
+      beforeEach(() => {
+        config = {
+          server: 'localhost',
+          options: {
+            database: 'master'
+          }
+        }
+        if (version === '3.0.0') {
+          config.userName = MSSQL_USERNAME
+          config.password = MSSQL_PASSWORD
+        } else {
+          config.authentication = {
+            options: {
+              userName: MSSQL_USERNAME,
+              password: MSSQL_PASSWORD
+            },
+            type: 'default'
+          }
+        }
+        return agent.load(plugin, 'tedious').then(() => {
+          tds = require(`../../../versions/tedious@${version}`).get()
+        })
+      })
+
+      after(() => {
+        return agent.close()
+      })
+
+      beforeEach((done) => {
+        connection = new tds.Connection(config)
+          .on('connect', err => done(err))
+      })
+
+      afterEach(() => {
+        connection.close()
+      })
+
+      it('should propagate context to callbacks', done => {
+        if (process.env.DD_CONTEXT_PROPAGATION === 'false') return done()
+
+        const span = tracer.startSpan('test')
+
+        tracer.scope().activate(span, () => {
+          const span = tracer.scope().active()
+          const request = new tds.Request('SELECT 1 + 1 AS solution', (err) => {
+            expect(tracer.scope().active()).to.equal(span)
+            done(err)
+          })
+          connection.execSql(request)
+        })
+      })
+
+      it('should run the callback in the parent context', done => {
```

Comment:
This test is basically testing the same thing as the previous one, but the previous one has the right implementation and this one has the correct name.